client/src/components/Modals/ModalDocument.vue

```diff
@@ -1,8 +1,15 @@
 <template>
   <modal @close="onClickClose">
-    <div slot="header">
-      <a :href="url" target="_blank"><h3>{{ title }}</h3></a>
-      <h5 class="m-0">{{ doi }}</h5>
+    <div class="flex-grow-1" slot="header">
+      <a :href="url" target="_blank">
+        <h3>{{ title }}</h3>
+      </a>
+      <div class="d-flex align-items-center justify-content-between">
+        <h5 class="m-0">{{ doi }}</h5>
+        <button class="btn btn-primary" @click="openKnowledgeView(true)">
+          Search Document Artifacts
```

Comment:
I think we probably want to say something like `Search Document in the context of the Knowledge space` and remove the link below.